packages/protocol/test/automata-attestation/common/AttestationTestBase.t.sol

```diff
@@ -0,0 +1,75 @@
+
+    function intialSetup() public {
+        // pinned September 23rd, 2023, 0221 UTC
+        // comment this line out if you are replacing sampleQuote with your own
+        // this line is needed to bypass expiry reverts for stale quotes
+        vm.warp(1_695_435_682);
+
+        vm.deal(admin, 100 ether);
+
+        vm.startPrank(admin);
+        p256Verifier = new P256Verifier();
+        sigVerifyLib = new SigVerifyLib(address(p256Verifier));
+        pemCertChainLib = new PEMCertChainLib();
+        attestation = new AutomataDcapV3Attestation(address(sigVerifyLib), address(pemCertChainLib));
+        attestation.setMrEnclave(mrEnclave, true);
+        attestation.setMrSigner(mrSigner, true);
+
+        string memory tcbInfoJson = vm.readFile(string.concat(vm.projectRoot(), tcbInfoPath));
+        string memory enclaveIdJson = vm.readFile(string.concat(vm.projectRoot(), idPath));
+
+        string memory fmspc = "00606a000000";
+        (bool tcbParsedSuccess, TCBInfoStruct.TCBInfo memory parsedTcbInfo) =
+            parseTcbInfoJson(tcbInfoJson);
+        require(tcbParsedSuccess, "tcb parsed failed");
+        attestation.configureTcbInfoJson(fmspc, parsedTcbInfo);
+
+        (bool qeIdParsedSuccess, EnclaveIdStruct.EnclaveId memory parsedEnclaveId) =
+            parseEnclaveIdentityJson(enclaveIdJson);
+        require(qeIdParsedSuccess, "qeid parsed failed");
+        attestation.configureQeIdentityJson(parsedEnclaveId);
```

Comment:
Nice. Can this code also be shared with the code in `SetDcapParams`? I know it's much less that can be shared, but still having the same setup code for the actual smart contract calls between the real deployment and tests seems very useful.

Reply:
Added a common base. 
Please @smtmfft can you have a look too ? :)